artifactory/commands/nuget/nugetconfig.go

```diff
@@ -0,0 +1,60 @@
+package nuget
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-go/artifactory/utils/prompt"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+)
+
+func CreateBuildConfig(global bool, confType utils.ProjectType) error {
+	projectDir, err := utils.GetProjectDir(global)
+	if err != nil {
+		return err
+	}
+	err = fileutils.CreateDirIfNotExist(projectDir)
+	if err != nil {
+		return err
+	}
+
+	configFilePath := filepath.Join(projectDir, confType.String()+".yaml")
+	if err := prompt.VerifyConfigFile(configFilePath); err != nil {
+		return err
+	}
+
+	var vConfig *viper.Viper
+	configResult := &ConfigFile{}
+	configResult.Version = prompt.BUILD_CONF_VERSION
+	configResult.ConfigType = confType.String()
+	configResult.Resolver.ServerId, vConfig, err = prompt.ReadServerId()
+	if err != nil {
+		return err
+	}
```

Comment:
This code is repeated at all native commands. Please use a helper function or use inheritance.

Reply:
Fixed